### DISCUSSION TITLE
problème de training  sur yolov5

### DISCUSSION THREAD

**Discussion by chaima2143** [ultralytics/yolov5#11311] [Q&A]
Starting training for 100 epochs...

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
  0% 0/97 [00:06<?, ?it/s]
Traceback (most recent call last):
  File "/content/yolov5/train.py", line 543, in <module>
    train(hyp, opt, device, tb_writer)
  File "/content/yolov5/train.py", line 304, in train
    loss, loss_items = compute_loss(pred, targets.to(device))  # loss scaled by batch_size
  File "/content/yolov5/utils/loss.py", line 117, in __call__
    tcls, tbox, indices, anchors = self.build_targets(p, targets)  # targets
  File "/content/yolov5/utils/loss.py", line 211, in build_targets
    indices.append((b, a, gj.clamp_(0, gain[3] - 1), gi.clamp_(0, gain[2] - 1)))  # image, anchor, grid indices
RuntimeError: result type Float can't be cast to the desired output type long int

**Comment by kronee0516** [comment]
seems it is cause by the the newer version of pytorch and older version of yolov5 content
took the last message to google
follow the instruction to make some changes on utils/loss.py
it would work , but i don't know if  it is  the right way to solve it
